apps/prairielearn/src/ee/routers/lti13.ts

```diff
@@ -0,0 +1,58 @@
+import { Router } from 'express';
+import lti13InstanceRouter from '../pages/lti13Instance/lti13Instance';
+import { Lti13InstanceSchema } from '../../lib/db-types';
+import asyncHandler = require('express-async-handler');
+import logRequest = require('../../middlewares/logRequest');
+import { features } from '../../lib/features';
+
+import { loadSqlEquiv, queryOptionalRow } from '@prairielearn/postgres';
+
+const sql = loadSqlEquiv(__filename);
+const router = Router({ mergeParams: true });
+
+router.use(
+  asyncHandler(async (req, res, next) => {
+    // The navbar relies on this property.
+    res.locals.urlPrefix = req.baseUrl;
+    next();
+  }),
+);
+
+router.use(logRequest);
```

Comment:
Is this necessary? The request should already be logged by the same middleware attached to the server as a whole.

Reply:
Yes, because the auth routes do their own CSRF, the `pl/lti13_instance` routes come before those middleware lines in server.js.

Reply:
I'm wondering if the right thing to do isn't to move `logRequest` earlier into the middleware chain. Tagging in @mwest1066 - is there a good reason that we haven't been logging certain requests?

Reply:
Filed issue #8691 so I'll remove it from here.

Reply:
Left a comment on that issue. I agree that we can move `logRequest` up to be just after `logResponse`.